google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/AppEngineInitializationComponent.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.appengine;
+
+import com.google.cloud.tools.intellij.startup.ConflictingAppEnginePluginCheck;
+
+import com.intellij.openapi.components.ApplicationComponent;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Performs App-Engine-related initialization operations.
+ */
+public class AppEngineInitializationComponent implements ApplicationComponent {
+
+  private static final String COMPONENT_NAME = "AppEngine.InitializationComponent";
+
+  @Override
+  public void initComponent() {
+    new ConflictingAppEnginePluginCheck().notifyIfConflicting();
```

Comment:
That was how I originally had it, but this was done in response to earlier comments on this PR. I think @patflynn was suggesting using a more common InitializationComponent for the AppEngine package/domain.